memory/Makefile

```diff
@@ -23,5 +23,5 @@ deploy: build
 	! [ -z $(NAMESPACE) ]  # call this like: make deploy NAMESPACE=default
 	kubectl -n $(NAMESPACE) apply -f service-account.yaml
 	python3 ../ci/jinja2_render.py '{"default_ns":{"name":"$(NAMESPACE)"}}' service-account-batch-pods.yaml service-account-batch-pods.yaml.out
```

Comment:
Hmm. This file doesn't exist. It seems it was mistakenly deleted by https://github.com/hail-is/hail/pull/9718/files

Reply:
Huh. I have no idea how the memory service works in PRs since that PR also removed the service account for memory.

Reply:
Ahah, the file is called service-account.yaml

Reply:
We shouldn't do this line at all, we never use the output!